forge-gui/res/adventure/Shandalar/decks/furnacetormentor.json

```diff
@@ -0,0 +1,12 @@
+{                                    
+"name":"Furnace Tormentor",
+  "template":
+  {
+   "count":60,
+   "colors":["Red, Green"],
```

Comment:
Hmm I've played against this enemy and it had a colorless deck 
![image](https://user-images.githubusercontent.com/67333662/232195749-a53c1969-a210-4f8e-8627-5d03300c15e6.png)

Reply:
Missing quotes in colors.
"Colors": ["Red", "Green"]